README.md

```diff
@@ -102,8 +102,7 @@ Create the BigQuery repository
 
          pushd ~/projects/PerfKitExplorer/data/samples_mart
 
-         bq load --project=MY_PROJECT_ID \
-           --source_format=NEWLINE_DELIMITED_JSON \
+         bq load --source_format=NEWLINE_DELIMITED_JSON \
```

Comment:
This parameter was deprecated in favor of --project_id, and should be modified instead of removed.